client/src/components/ProfilePhoto/ProfilePhoto.tsx

```diff
@@ -1,11 +1,58 @@
+import axios from 'axios';
 import { useAuth } from '../../context/useAuthContext';
+import { useState } from 'react';
 import useStyles from './useStyles';
-import { Button, Card, Typography, Avatar, CardActions, Paper } from '@material-ui/core';
+import { Button, Card, Typography, Avatar, CardActions, Paper, Input, FormControlLabel } from '@material-ui/core';
 import DeleteIcon from '@material-ui/icons/Delete';
+import { Formik } from 'formik';
 
-const ProfilePhoto = (): JSX.Element => {
+interface MyFormValues {
+  fileNames: string;
+}
+
+const ProfilePhoto = () => {
   const classes = useStyles();
+  const [file, setFile] = useState('');
+  const [filename, setFilename] = useState(' Upload a file from your device!!!!');
+  const [uploadedFile, setUploadedFile] = useState({});
+  const [isInputClicked, setIsInputClicked] = useState(false);
+
   const { loggedInUser } = useAuth();
+  const [avatarImage, setAvatarImage] = useState(`https://robohash.org/${loggedInUser!.email}.png`);
+
+  const onFormSubmit = (values: any, actions: any) => {
+    //
+  };
+  const initialValues: MyFormValues = { fileNames: '' };
+
+  const onInputChange = (e: any) => {
+    setFile(e.target.files[0]);
+    setFilename(`Upload ${e.target.files[0].name}`);
+    setIsInputClicked(true);
+  };
+
+  const handleSubmit = async (e: any) => {
+    e.preventDefault();
+    const formData = new FormData();
+    formData.append('file', file);
+    try {
+      const res = await axios.post('http://localhost:3001/upload', formData, {
```

Comment:
Maybe create this as a helper function and move it to helpers/API, so that we can stay consistent with the code organization/structure.

Reply:
created helper function formDataAPI.ts